src/mudlet.cpp

```diff
@@ -2088,6 +2092,38 @@ bool mudlet::createBuffer(Host* pHost, const QString& name)
     return false;
 }
 
+bool mudlet::setProfileStyleSheet(Host* pHost, const QString& styleSheet)
+{
+    if (!pHost || !pHost->mpConsole) {
+        return false;
+    } else {
+        pHost->mProfileStyleSheet = styleSheet;
+        pHost->mpConsole->setStyleSheet(styleSheet);
+        pHost->mpEditorDialog->setStyleSheet(styleSheet);
+
+        if (pHost->mpNotePad) {
+            pHost->mpNotePad->setStyleSheet(styleSheet);
+            pHost->mpNotePad->notesEdit->setStyleSheet(styleSheet);
+        }
+        if (pHost->mpDockableMapWidget) {
+            pHost->mpDockableMapWidget->setStyleSheet(styleSheet);
+        }
+
+        QMutableMapIterator<QString, TDockWidget*> itDockWidget(pHost->mpConsole->mDockWidgetMap);
```

Comment:
I know we use the same old-style code elsewhere - but with modern c++ it can be much simpler -

```cpp
   for (auto& dockWidget : pHost->mpConsole->mDockWidgetMap) {
     dockWidget.setStyleSheet(styleSheet);
   }
```

Something like that off the top of my head :)